packages/hyperion-autologging/src/AutoLogging.ts

```diff
@@ -71,7 +71,7 @@ export function init(options: InitOptions): boolean {
     setComponentNameValidator(options.componentNameValidator);
   }
 
-  if (global instanceof Window && global?.document.createElement != null) {
+  if (global && (global as Window)?.document?.createElement != null) {
```

Comment:
This is what I was talking about the other day as well.
To be fully safe, you should do it this way:
```
if (typeof global !== 'undefined' && (global as Window)?.document?.createElement != null)
```

Reply:
Ah,  thanks.  Done in latest push.